<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreationContact</class>
 <widget class="QWidget" name="CreationContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="layout_champs">
     <item>
      <layout class="QVBoxLayout" name="layout_photo">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="image">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_importerImage">
         <property name="text">
          <string>Importer Photo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_nom_prenom">
       <item>
        <layout class="QHBoxLayout" name="layout_nom">
         <item>
          <widget class="QLabel" name="l_nom_2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Nom* :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_nom"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_prenom">
         <item>
          <widget class="QLabel" name="l_prenom">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Prénom* : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_prenom"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_infos">
       <item>
        <layout class="QHBoxLayout" name="layout_tel">
         <item>
          <widget class="QLabel" name="l_tel">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Téléphone: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_tel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_mail">
         <item>
          <widget class="QLabel" name="l_mail">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Mail* : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_mail"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="l_entreprise">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Entreprise: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_entreprise"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="l_obligation">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>* : Obligatoire</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_infoText">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_buttons">
     <item>
      <widget class="QPushButton" name="b_annuler">
       <property name="whatsThis">
        <string>Annuler la création d'un contact</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_creer">
       <property name="text">
        <string>Créer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
